src/main/java/core/basesyntax/strategy/OperationStrategyImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitOperation;
+import core.basesyntax.strategy.operation.OperationHandler;
+import java.util.Map;
+
+public class OperationStrategyImpl implements OperationStrategy {
```

Comment:
According to the SOLID principles, particularly the Dependency Inversion Principle, it's better to depend on abstractions rather than concrete classes. Consider using an interface for the map declaration if possible, for example `Map<FruitOperation.Operation, ? extends OperationHandler>`.